docs/changes.rst

```diff
@@ -90,6 +90,10 @@ Released: not yet
   An existing file with the old name is migrated automatically.
   (See issue #716)
 
+* Refactor error handling for Connection load if there are problems with the
+  YAML file Creaties a unit test and function error test. (see issue #661)
```

Comment:
Typo "Creaties". Missing dot before that.

Reply:
DONE